monitors/index.js

```diff
@@ -223,69 +213,47 @@ async function Poll(callback) {
-    //    address: "0x12345",
-    //    crAlert: 1.50 }, // Note 150% is represented as 1.5
-    //  ...];
-    const walletMonitorObject = JSON.parse(process.env.WALLET_MONITOR_OBJECT);
-
-    // Contract monitor. The monitor needs the addresses of the liquidator and disute bots to inform logs. EG:
-    // { "monitoredLiquidators": ["0x1234","0x5678"],
-    //   "monitoredDisputers": ["0x1234","0x5678"] }
-    const contractMonitorObject = JSON.parse(process.env.CONTRACT_MONITOR_OBJECT);
-
-    // Synthetic Peg monitor. Specify the deviationAlertThreshold, volatilityWindow and volatilityAlertThreshold. EG:
-    // { "deviationAlertThreshold": 0.5, // if the deviation in token price exceeds this value an alert is fired.
-    //   "volatilityWindow": 600 // Length of time (in seconds) to snapshot volatility.
-    //   "volatilityAlertThreshold": 0.1 } // Error threshold for pricefeed's price volatility over `volatilityWindow`.
-    const syntheticPegMonitorObject = JSON.parse(process.env.SYNTHETIC_PEG_MONITOR_OBJECT);
-
-    if (!process.env.UNISWAP_PRICE_FEED_CONFIG || !process.env.MEDIANIZER_PRICE_FEED_CONFIG) {
-      throw new Error(
-        "Bad input arg! Specify `PRICE_FEED_CONFIG` and `MEDIANIZER_PRICE_FEED_CONFIG` to define the price feed settings."
-      );
-    }
+    // Monitor config contains all configuration settings for all monitor modules. This includes the following:
+    // MONITOR_CONFIG={
+    //  "botsToMonitor": [{ name: "Liquidator Bot",       // friendly bot name
+    //     address: "0x12345"                             // bot address
+    //    "collateralThreshold": "500000000000000000000", // 500e18 collateral token currency.
+    //    "syntheticThreshold": "2000000000000000000000", // 200e18 synthetic token currency.
+    //    "etherThreshold": "500000000000000000" },       //0.5e18 Wei alert
+    //  ...],
+    //  "walletsToMonitor": [{ name: "Market Making bot",  // friendly bot name
+    //    address: "0x12345",                             // bot address
+    //    crAlert: 1.50 },                                // CR monitoring threshold. 1.5=150%
+    //  ...],
+    //  "monitoredLiquidators": ["0x1234","0x5678"],       // array of liquidator bots of interest.
+    //  "monitoredDisputers": ["0x1234","0x5678"],         // array of disputer bots of interest.
+    //  "deviationAlertThreshold": 0.5,                    // if deviation in token price exceeds this fire alert.
+    //  "volatilityWindow": 600,                           // Length of time (in seconds) to snapshot volatility.
+    //  "pegVolatilityAlertThreshold": 0.1,                // Threshold for synthetic peg (identifier) price volatility over `volatilityWindow`.
+    //  "syntheticVolatilityAlertThreshold": 0.1,           // Threshold for synthetic token on uniswap price volatility over `volatilityWindow`.
+    // }
+    const monitorConfig = process.env.MONITOR_CONFIG ? JSON.parse(process.env.MONITOR_CONFIG) : null;
```

Comment:
note only ONE config object containing all configs within it. this makes it much easier to parameterized as all inputs are optional.

Reply:
Yeah this is a lot nicer to collapse `botMonitorObject`, `walletMonitorObject`, `contractMonitorObject`, and `syntheticPegMonitorObject` into one object